build/pbm-entry.sh

```diff
@@ -24,13 +24,13 @@ if [ "${1:0:9}" = "pbm-agent" ]; then
 
 			# check in case if shard has role 'shardsrv'
 			set +o xtrace
-			mongo ${PBM_MONGO_OPTS} "${PBM_MONGODB_URI}" --eval="db.isMaster().\$configServerState.opTime.ts" --quiet | tee "$OUT"
+			mongosh ${PBM_MONGO_OPTS} "${PBM_MONGODB_URI}" --eval="db.isMaster().\$configServerState.opTime.ts" --quiet | tee "$OUT"
```

Comment:
I think we do not have mongosh in pbm image right now

Reply:
Yes, I can confirm that we don't. What can we do about it?